### PR TITLE
Update PackNameConverter.cs

### PR DESCRIPTION
**Pull request by Mayot**
Add Madness at the Darkmoon Faire.

### CODE REVIEW

**Review by Mayot — COMMENTED** [review]
looks good to go

File: PackHistorian/View/PackNameConverter.cs [hunk 1]
```diff
@@ -328,6 +328,26 @@ class PackNameConverter : IValueConverter {
...
+          { Locale.zhTW, "Golden Pack" },
+          { Locale.zhCN, "Golden Pack" },
+          { Locale.itIT, "Golden Pack" },
+          { Locale.ptBR, "Golden Pack" },
+          { Locale.plPL, "Golden Pack" },
+          { Locale.ptPT, "Golden Pack" },
+          { Locale.jaJP, "Golden Pack" },
+          { Locale.thTH, "Golden Pack" },
+        }
+      },
```
Comment (TheMrKochan) [inline PackHistorian/View/PackNameConverter.cs:350]: 470 number is not exactly a "Golden pack". This is a special pack of Hunter cards.

**Comment by TheMrKochan** [comment]
498 - Year of the Dragon, 545 - special pack of Mage cards :)

**Comment by TheMrKochan** [comment]
Wait a second! 470 - special pack of Hunter cards.

**Pull request merged**